src/pymatgen/io/abinit/abiobjects.py

```diff
@@ -56,7 +56,7 @@ def lattice_from_abivars(cls=None, *args, **kwargs):
             raise ValueError(f"The sum of angdeg must be lower than 360, {ang_deg=}")
 
         # This code follows the implementation in ingeo.F90
-        # See also http://www.abinit.org/doc/helpfiles/for-v7.8/input_variables/varbas.html#angdeg
```

Comment:
I have to replace this with the latest doc instead of the original v7.8, because the original URL seems inaccessible.